deployments/polygon/usdc/migrations/1675200105_raise_polygon_supply_caps.ts

```diff
@@ -0,0 +1,108 @@
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { exp, proposal } from '../../../../src/deploy';
+import { expect } from 'chai';
+
+export default migration('1675200105_raise_polygon_supply_caps', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, governanceDeploymentManager: DeploymentManager) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+    const { utils } = ethers;
+
+    const {
+      fxRoot,
+      governor
+    } = await governanceDeploymentManager.getContracts();
+
+    const {
+      bridgeReceiver,
+      comet,
+      cometAdmin,
+      configurator,
+      WBTC,
+      WETH,
+      WMATIC
+    } = await deploymentManager.getContracts();
+
+    const wbtcCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address', 'uint128'],
+      [comet.address, WBTC.address, exp(10_000, 8)] // XXX add actual amount
+    );
+    const wethCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address', 'uint128'],
+      [comet.address, WETH.address, exp(10_000, 18)] // XXX add actual amount
+    );
+    const wmaticCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address', 'uint128'],
+      [comet.address, WMATIC.address, exp(10_000, 18)] // XXX add actual amount
+    );
+    const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [configurator.address, comet.address]
+    );
```

Comment:
I think we should stick to the inline list with comments for each action format on these on all the other proposal, this format requires a lot of jumping around to read